src/qibolab/instruments/qm/sequence.py

```diff
@@ -68,7 +68,8 @@ class QMPulse:
     def __post_init__(self):
         pulse_type = self.pulse.type.name.lower()
         amplitude = format(self.pulse.amplitude, ".6f").rstrip("0").rstrip(".")
-        self.element: str = f"{pulse_type}{self.pulse.qubit}"
+        if self.element is None:
+            self.element = f"{pulse_type}{self.pulse.qubit}"
```

Comment:
Btw, does quantum machine require a string as element identifier?

It could well be. But if not, maybe just put the pair.

Reply:
I tried using a tuple and I got the following error:
```
qm.exceptions.ConfigValidationException: {'elements': defaultdict(<class 'dict'>, {('drive', 0): {'key': ['Not a valid string.']}, ('drive', 2): {'key': ['Not a valid string.']}, ('drive', 3): {'key': ['Not a valid string.']}})}
```
so it seems it requires a string. List is also not an option as this is the key of a dict.

In 0.2 this will probably be replaced by the channel name and it will hopefully be more readable and not hard-coded. For now I will leave it as it is.

Reply:
Ah, ok, because it is most likely serializing it as JSON. In principle, we could decide to use as a string the JSON serialization of the element object (when we'll have a serializable one with Pydantic), but let's keep this for 0.2, consistently with the channels upgrade :)